<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>909</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1222</width>
    <height>909</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1222</width>
    <height>909</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3DMeshMetric</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxVisualization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>DATAS</string>
    </property>
    <widget class="QLabel" name="labelLoaded">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Loaded Files :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>delete all files</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetLoadedMesh">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisplayAll">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>display all files</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>add file(s)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeleteOne">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>delete selected file</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHideAll">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>hide all files</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCamera">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>CAMERA</string>
    </property>
    <widget class="QPushButton" name="pushButtonLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRight">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFront">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ante</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Post</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUp">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Sup</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDown">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Inf</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaMesh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>721</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>711</width>
      <height>751</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>719</width>
       <height>749</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidgetVisualization">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabVisualization">
     <attribute name="title">
      <string>Visualization</string>
     </attribute>
     <widget class="QRadioButton" name="radioButtonPoints">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOpacity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Opacity :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSurface">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>109</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Surface</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxError">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>(un)display distance error</string>
      </property>
      <property name="text">
       <string>Display Error</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>change color of selected file</string>
      </property>
      <property name="text">
       <string>Change Color</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonWireframe">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>109</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Wireframe</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderOpacity">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>change opacity of selected file</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberOpacity">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>value of opacity</string>
      </property>
      <property name="numDigits">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonResetAll">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Reload File</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSmoothing">
     <attribute name="title">
      <string>Filters</string>
     </attribute>
     <widget class="QLabel" name="labelSmoothing">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Smoothing : ( Number of Iteration )</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDecimate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Down-Sampling : ( Percentage lost )</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRunDecimate">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxDecimate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRunSmoothing">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxIteration">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>20000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidgetError">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabDistance">
     <attribute name="title">
      <string>Distance</string>
     </attribute>
     <widget class="QLabel" name="labelMeshB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh B</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxMeshB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>choose 2nd file for distance error</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonApply">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMeshA">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelMeshA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonAbsoluteDistance">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Absolute Distance</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSignedDistance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Signed Distance</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>93</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabParameters">
     <attribute name="title">
      <string>Advanced Parameters</string>
     </attribute>
     <widget class="QLabel" name="labelMinSampFreq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Sampling Frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSampStep">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sampling Step</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSamplingStep">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMinSampFreq">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>56</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabResults">
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonUpdateColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Update color</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRange_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>  ]</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRange_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  ,</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRange">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Points Range :  [</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMax">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="ColormapWidget" name="widgetColor" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>238</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditA">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditB">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>63</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>From : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>To :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1222</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddNewFile"/>
    <addaction name="actionAddNewRepository"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionAddNewFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new file</string>
   </property>
  </action>
  <action name="actionAddNewRepository">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new repository</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColormapWidget</class>
   <extends>QWidget</extends>
   <header location="global">ColormapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
